internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -441,3 +441,30 @@ func (o *OCMProvider) CreateMachinePool(request *types.MachinePoolRequest) (*typ
 
 	return request, err
 }
+
+// GetQuotaCosts returns a list of ocm resource quota cost information for the authenticated user
+func (o *OCMProvider) GetQuotaCosts() ([]types.QuotaCost, error) {
+	var quotaCostList []types.QuotaCost
+	account, err := o.ocmClient.GetCurrentAccount()
+	if err != nil {
+		return quotaCostList, err
+	}
+	orgID := account.Organization().ID()
```

Comment:
As a note:
OCM types usually provide two types of getters for the attributes of an object:
* A getter with the name of the attribute
* A getter Get<AttributeName>

For example, for organization of an AMS account you have the methods:
* Organization()
* GetOrganization()

The second type of getter is useful because it allows you to check whether the attribute has been set or not, and also allows differentiating between the zero value of the type and whether it's been set or not for the cases where the zero value is an accepted value.

Reply:
Good to know. I was always wondering what's the difference between the two.

Reply:
I changed it to use GetId() instead so we wont have to check the value of the id :+1: